### ISSUE TITLE
Capitalization for LCSH and Homosaurus [Lower priority]

### ISSUE THREAD

**Issue by kjrawson** [CollegeOfTheHolyCross/dta_sufia#116]
-yes, it is true that capitalization for the different vocabularies isn't standardized in our system. Also, when you are browsing the topic facet and you sort by A-Z, there display as two separate lists (e.g., it goes A-Z for one vocabulary, the for the other). But, I would wait on this––some of the other tickets are higher priority. I'll add a ticket for it so that we don't forget.

**Comment by scande3** [comment]
Standardization of capitalization of terms should be complete now.

**Comment by kjrawson** [comment]
Looks great! Thanks so much, Steven! The new Homosaurus will follow LCSH capitalization logic, FYI.
